airflow/providers/common/sql/hooks/sql.py

```diff
@@ -264,15 +266,39 @@ def last_description(self) -> Sequence[Sequence] | None:
             return None
         return self.descriptions[-1]
 
+    @overload
+    def run(
+        self,
+        sql: str | Iterable[str],
+        autocommit: bool = ...,
```

Comment:
Let’s copy the real defaults since they are quite simple. This would be much friendlier to editor popups.